### DISCUSSION TITLE
GST VITS output on blizzard dataset is not comming as expected.

### DISCUSSION THREAD

**Discussion by manmay-nakhashi** [coqui-ai/TTS#1901] [General Q&A]
i was trying https://github.com/coqui-ai/TTS/tree/dev-emotion-rebased branch for GST based vits training. i have found that speaker styles are not adapted even after 1M steps. 
@Edresson can you give me some suggestions on how to improve that ?

**Comment by Edresson** [comment]
Hi @manmay-nakhashi, 

The official branch is "dev" and the implementation available in other branches are experiments and has no guarantee that it will work. I worked with GST on the branch [dev-emotion-rebased](https://github.com/coqui-ai/TTS/tree/dev-emotion-rebased)  some time ago but could not achieve a good result then I stopped these experiments.

Best regarding,
Edresson

**Reply by manmay-nakhashi** [reply]
@Edresson does capacitron gives better output then GST?
